app/Http/Services/Tabbar/TabbarService.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace App\Http\Services\Tabbar;
+
+use Illuminate\Http\Request;
+use App\Http\PathInfo;
+
+/**
+ * TabbarService
+ *
+ * タブバーの加工を行う
+ *
+ */
+class TabbarService {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition